### PR TITLE
Update isq.ttl

### PR DESCRIPTION
**Pull request by jsimonclark**
The PR adds quantities and physical dimensions from the ISQ/IEC 80000 standard that are needed for the domain-electrochemistry and domain-battery ontologies.

### CODE REVIEW

File: disciplines/isq.ttl [hunk 1]
```diff
@@ -90,6 +108,30 @@ email: emanuele.ghedini@unibo.it"""@en ,
...
+###  http://emmo.info/emmo#EMMO_03d32783_3780_43c3_8b73_7efe9aeeed54
+:EMMO_03d32783_3780_43c3_8b73_7efe9aeeed54 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_b3600e73_3e05_479d_9714_c041c3acf5cc
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-1.2" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "fundamental translation vectors for the crystal lattice"@en ;
+                                           skos:prefLabel "FundamentalLatticeVectors"@en .
```
Comment (jesper-friis) [inline disciplines/isq.ttl:132]: Preferred labels should be singular noun, so a better prefLabel would be "FundamentalLatticeVector"

File: disciplines/isq.ttl [hunk 2]
```diff
@@ -90,6 +108,30 @@ email: emanuele.ghedini@unibo.it"""@en ,
...
+
+###  http://emmo.info/emmo#EMMO_03d32783_3780_43c3_8b73_7efe9aeeed54
+:EMMO_03d32783_3780_43c3_8b73_7efe9aeeed54 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_b3600e73_3e05_479d_9714_c041c3acf5cc
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-1.2" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "fundamental translation vectors for the crystal lattice"@en ;
```
Comment (jesper-friis) [inline disciplines/isq.ttl:131]:
```suggestion
                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "fundamental translation vector for the crystal lattice"@en ;
```

File: disciplines/isq.ttl [hunk 3]
```diff
@@ -256,6 +477,39 @@ email: emanuele.ghedini@unibo.it"""@en ,
...
+###  http://emmo.info/emmo#EMMO_134b78a0_63f8_4f7f_8098_925d1d934e48
+:EMMO_134b78a0_63f8_4f7f_8098_925d1d934e48 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_608a1b30_df6f_4bbb_9dc3_5c0de92fd9cf
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-2.2" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "fundamental translation vectors for the reciprocal lattice"@en ;
+                                           skos:prefLabel "FundamentalReciprocalLatticeVectors"@en .
```
Comment (jesper-friis) [inline disciplines/isq.ttl:500]: Suggest to change to singular: FundamentalReciprocalLatticeVector

File: disciplines/isq.ttl [hunk 4]
```diff
@@ -626,6 +1401,83 @@ Entropy"""@en ;
...
+
+
+###  http://emmo.info/emmo#EMMO_47f2da35_3b01_4290_964a_589c593e85c6
+:EMMO_47f2da35_3b01_4290_964a_589c593e85c6 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_dae32a4a_d8da_4047_81b0_36a9713fdce1 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_3ecff38b_b3cf_4a78_b49f_8580abf8715b
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "5-23" ;
```
Comment (jesper-friis) [inline disciplines/isq.ttl:1455]: Do you have an elucidation for PlanckFunction?

File: disciplines/isq.ttl [hunk 5]
```diff
@@ -636,6 +1488,17 @@ Entropy"""@en ;
...
 
+###  http://emmo.info/emmo#EMMO_4c3ccf65_0950_4328_b04b_91d1eaa14681
+:EMMO_4c3ccf65_0950_4328_b04b_91d1eaa14681 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_a06e3d38_1aa8_4f45_89a8_722dbacfda24 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_3227b821_26a5_4c7c_9c01_5c24483e0bd0
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "9-40.2" ;
+                                           skos:prefLabel "ThermalDiffusionFactor"@en .
```
Comment (jesper-friis) [inline disciplines/isq.ttl:1499]: Missing elucidation

**Review by emanueleghedini — CHANGES_REQUESTED** [review]
Add the new content in a separate module, non modifying the isq.ttl, to facilitate reviewing.
Place the new classes under ISQ base or derived quantity classes also.

**Comment by jsimonclark** [comment]
I have added the new content in a separate module (isq_bigmap.ttl) and classified everything under ISQ base or derived quantities.

File: disciplines/isq.ttl (hunk 1 above)
Comment (jsimonclark) [reply disciplines/isq.ttl]: done

File: disciplines/isq.ttl (hunk 2 above)
Comment (jsimonclark) [reply disciplines/isq.ttl]: done

File: disciplines/isq.ttl (hunk 4 above)
Comment (jsimonclark) [reply disciplines/isq.ttl]: done

File: disciplines/isq.ttl (hunk 5 above)
Comment (jsimonclark) [reply disciplines/isq.ttl]: done

File: disciplines/isq.ttl (hunk 3 above)
Comment (jsimonclark) [reply disciplines/isq.ttl]: done

**Comment by jesper-friis** [comment]
This is a huge and great piece of work! I have not yet gone through everything in detail, but here are some comments:
- Duration should be a subclass of Time (not a new base quantity). If QUDT has defined duration, we could add a reference to it
- CarrierLifeTime should be a subclass of TimeConstant (see Wikidata)
- From the elucidations of Displacement and Distance, it seems that Distance is the norm of the Displacement. Maybe we could add a comment about that.
- Time appears bold, as it has been modified. Has it been added twice?
- According to the Gold Book, Solubility can be measured as concentration, molarity, mole fraction, etc... Hence, it should not be a subclass of AmountConcentration.  
- We could add Concentration (https://en.wikipedia.org/wiki/Concentration) as a superclass for AmountConcentration, MassConcentration, VolumeConcentration, ...
- PhaseSpeedOfElectromagneticWaves is a scalar. Hence, it should be a subclass of Speed instead of Velocity.

For discussion with Emanuele:
- Under CategorizedPhysicalQuantity, should we introduce ScalarQuantity, VectorQuantity, Rank2TensorQuantity, ... and use them to ontologically distinguish between quantities like Displacement/Distance and Velocity/Speed, etc? 
- Under CategorizedPhysicalQuantity, would it make sense to add a superclass for quantities like Solubility and Composition that can be expressed in terms of other quantities with different physical dimensions? What would be a good label for such a superclass? UndefinedQuantity or OpenQuantity? Introdusing this superclass would be useful for automated checking of physical quantities - they should either have a physical dimension or be an UndefinedQuantity. 
-

**Comment by jsimonclark** [comment]
I have added the units definitions to this PR, because they also require the same PhysicalDimension definitions as the quantities.

**Pull request closed without merging**